### PR TITLE
Update ephemeral-volumes.md

### PR DESCRIPTION
**Pull request by akshayamadhuri**
issue #41904

### CODE REVIEW

File: content/en/docs/concepts/storage/ephemeral-volumes.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ different purposes:
   kinds of Kubernetes data into a Pod
 - [CSI ephemeral volumes](#csi-ephemeral-volumes):
   similar to the previous volume kinds, but provided by special
-  [CSI drivers](https://github.com/container-storage-interface/spec/blob/master/spec.md)
+  [CSI drivers](https://kubernetes-csi.github.io/docs/drivers.html)
```
Comment (xing-yang) [inline content/en/docs/concepts/storage/ephemeral-volumes.md:51]: This change is not needed.  This doc is already referenced on line 52.

Comment (xing-yang) [reply content/en/docs/concepts/storage/ephemeral-volumes.md:51]: Instead, you can update the URL on line 52 with https://kubernetes-csi.github.io/docs/ephemeral-local-volumes.html.

Comment (akshayamadhuri) [reply content/en/docs/concepts/storage/ephemeral-volumes.md:51]: @xing-yang  opened PR

File: content/en/docs/concepts/storage/ephemeral-volumes.md [hunk 2]
```diff
@@ -48,7 +48,7 @@ different purposes:
...
   which specifically [support this feature](https://kubernetes-csi.github.io/docs/drivers.html)
```
Comment (AmarNathChary) [inline content/en/docs/concepts/storage/ephemeral-volumes.md:52]:
```suggestion
  which specifically [support this feature](https://kubernetes-csi.github.io/docs/ephemeral-local-volumes.html)
```

Comment (AmarNathChary) [reply content/en/docs/concepts/storage/ephemeral-volumes.md:52]: This change of link should be a proper link for it @akshayamadhuri WDYT?

Comment (akshayamadhuri) [reply content/en/docs/concepts/storage/ephemeral-volumes.md:52]: @AmarNathChary  i already raised another PR and its been merged i am closing this issue for now

**Pull request closed without merging**